README.md

```diff
@@ -23,16 +23,17 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
 + JDK 8
 + Apache Spark 2.2.0
 + Jupyter Notebook 4.1
-+ BigDL 0.3.0
++ BigDL 0.4.0
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
-### Start Jupyter Server
+### Start Jupyter Server and Toree Kernel
```

Comment:
don't change the title

Reply:
Add a new section after this one

### Run scala notebook on toree
* Run ```pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0/snapshots/dev1/toree-pip/toree-0.2.0.dev1.tar.gz```
* Run toree_install.sh
* jupyter notebook --notebook-dir=./ --ip=* --no-browser